build/mulesoft_discovery_agent.yml

```diff
@@ -1,35 +1,28 @@
 central:
-  agentName: ${CENTRAL_AGENTNAME:""}
-  organizationID: "<Amplify_ORG_ID>"
```

Comment:
someone just asked me about these, the <xxx> being confusing, thanks for cleaning this up